packages/api/src/pre-start/env-required.ts

```diff
@@ -15,6 +15,7 @@ const requiredEnvVariables = [
   'API_URL',
   'DISCORD_TOKEN',
   'DISCORD_GUILD_ID',
+  'DISCORD_BOT_API_KEY',
```

Comment:
Use instead the more generic new env fields:

```
API_KEYS=
API_KEY_ROLES=
```

The setup script will generate these keys when run. Bear in mind that more than one key can be defned in those fields. Use the key with the role `API_KEY_DISCORD_BOT`

https://github.com/givepraise/praise/blob/7ad7e1ed6aac1cb55e3ad629843dceacd9154ae8/packages/setup/src/index.ts#L162-L163